plugins/tiddlywiki/prosemirror/ast/from-prosemirror.js

```diff
@@ -21,24 +21,24 @@ function paragraph(builder, node) {
 }
 
 // Map ProseMirror mark types to HTML tags
-const markTypeMap = {
+var markTypeMap = {
```

Comment:
Since we allow ES2017 for v5.4.0 you can use `const` and `let` now

Reply:
OkOk, that would be much more fun.